raiden_libs/test/fixtures/contracts.py

```diff
@@ -54,13 +54,14 @@ def token_network_contract(
         secret_registry_contract,
         standard_token_contract
 ):
+    network_id = int(secret_registry_contract.web3.version.network)
     return deploy_tester_contract(
         'TokenNetwork',
         {
             'Token': standard_token_contract.address.encode(),
             'SecretRegistry': secret_registry_contract.address.encode()
         },
-        [standard_token_contract.address, secret_registry_contract.address]
+        [standard_token_contract.address, secret_registry_contract.address, network_id]
```

Comment:
How is this additional parameter handled in `deploy_tester_contract`? There are no changes there...

Reply:
these are constructor arguments, deploy will throw if they're incorrect

Reply:
But did this change? I don't  find changes in the contracts!?

Reply:
The change is from 23 days ago.. But maybe no test is using this fixture so it didn't cause problems.